sdk/textanalytics/Azure.AI.TextAnalytics/src/LinkedEntity.cs

```diff
@@ -7,6 +7,12 @@
 namespace Azure.AI.TextAnalytics
 {
     /// <summary>
+    /// A word or phrase identified from the input text as important, as well
```

Comment:
This is not accurate. The identified entity may not be "important" (that's one of the key differences between key phrases and entities). Instead perhaps rephrase to: `A word or a phrase identified as a known data source term. A link to the entry in the data source is specified as well as a formal name. Note that the formal name may differ from the exact text match in the input document.`

Reply:
Thanks Assaf!  Will update.